extensions/positron-python/python_files/positron/positron_ipykernel/inspectors.py

```diff
@@ -738,7 +738,7 @@ def get_display_value(
             "edgeitems": ARRAY_EDGEITEMS,
             "linewidth": print_width,
         }
-        options_obj = torch._tensor_str.PRINT_OPTS
+        options_obj = torch._tensor_str.PRINT_OPTS  # type: ignore[reportGeneralTypeIssues]
```

Comment:
these errors were generated in CI, but _tensor_str does seem to exist and is tested